shiny/www/shared/bslib/_version.json

```diff
@@ -1,5 +1,5 @@
 {
   "note!": "This file is auto-generated by scripts/htmlDependencies.R",
   "package": "bslib",
-  "version": "Github (rstudio/bslib@f05bd23d7df5a7465e418a5794925dacdd27bb6b)"
```

Comment:
This is version of bslib we previously used in the last released version of py-shiny.

Reply:
Now tagged at https://github.com/rstudio/bslib/releases/tag/py-shiny